content/support-training.html

```diff
@@ -0,0 +1,173 @@
+            Martina Ferrari <br/> 
+            (Independent Contractor)
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.metricfire.com/hosted-prometheus" class="btn btn-training btn-default darkgray-bg btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/metricfire.png" alt="MetricFire">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.robustperception.io/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/robust-perception.png" alt="Robust Perception">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          
+          <a href="https://www.robustperception.io/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/sysdig.svg" alt="Sysdig">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.weave.works/features/prometheus-monitoring/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/weaveworks.svg" alt="Weaveworks">
+          </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="container">
+      <div class="col-md-12 doc-content">
+        <a name="add"></a><h2>Add a resource</h2>
+        <p>
+          Anyone who has developed a Prometheus training program or offers related services
+          can add themselves to this page if they meet the following requirements.
+        </p>
+        </div>
+        
+        <div class="col-md-6 doc-content">
+          <!-- NOTE: WIP -->
+          <b>For courses and training programs:</b>
+          <ul>
+            <li>The course must have a publicly available syllabus or outline on the page linked.</li>
+            <li>The course must provide x numbers of course hours/work</li>
+            <li>If the course is instructor led, you must provide credentials (a link to your LinkedIn profile 
+              or resume will suffice)</li>
+          </ul>
+          
+        </div>
+        <div class="col-md-6 doc-content">
+          <b>For consultants or businesses:</b>
+          <ul>
+            <li>Consultants must provide credentials (a link to your LinkedIn profile or resume will suffice)</li>
+            <li>Businesses must outline what Prometheus-related services they provide on the page linked.</li>
+          </ul>
+          
+          <p>To submit, open a pull request against this page in the <code>prometheus/docs</code> repository. 
+            Include all the information requested in the PR description. Pull requests against
+            this page require 2 approvers from the Prometheus team before merging.</p>
```

Comment:
This is not consistent with our governance, and this level of detail doesn't belong here.

I personally see kinda it that if they're a member of the community involved enough to provide support/training/services they shouldn't have a problem finding where this file is to send a PR.

Reply:
I'm for listing the submission instructions, but would omit the sentence about approvals (we'll handle that internally according to the project's governance - lazy consensus or vote in case of conflict).

Reply:
👍